### PR TITLE
adds pt-BR localization for files/jsondata/L10n-*.json #8749

### PR DESCRIPTION
**Pull request by josielrocha**
### Description

Part of [mdn/translated-content#8749](https://github.com/mdn/translated-content/issues/8749)

### CODE REVIEW

File: files/jsondata/L10n-SVG.json [hunk 1]
```diff
@@ -85,20 +85,23 @@
...
     "es": "Cualquier número de elementos de animación, elementos descriptivos, {{SVGElement(\"mask\")}}, {{SVGElement(\"clipPath\")}}, {{SVGElement(\"marker\")}}, {{SVGElement(\"script\")}}, y {{SVGElement(\"style\")}}, junto con cero o un {{SVGElement(\"meshgradient\")}} elemento.",
     "fr": "N'importe quelle quantité d'éléments d'animation ou d'éléments descriptifs ou d'éléments {{SVGElement(\"mask\")}}, {{SVGElement(\"clipPath\")}}, {{SVGElement(\"marker\")}}, {{SVGElement(\"script\")}}, et {{SVGElement(\"style\")}}, ainsi que zéro ou un élément {{SVGElement(\"meshgradient\")}}.",
     "ja": "任意の数のアニメーション要素、説明的要素、 {{SVGElement(\"mask\")}}, {{SVGElement(\"clipPath\")}}, {{SVGElement(\"marker\")}}, {{SVGElement(\"script\")}}, {{SVGElement(\"style\")}} の各要素と 0 個または 1 個の {{SVGElement(\"meshgradient\")}} 要素。",
+    "pt-BR": "Qualquer número de elementos de animação, elementos descritivos, elementos {{SVGElement(\"mask\")}}, {{SVGElement(\"clipPath\")}}, {{SVGElement(\"marker\")}}, {{SVGElement(\"script\")}}, e elementos {{SVGElement(\"style\")}}, juntamente com zero ou um elemento {{SVGElement(\"meshgradient\")}}.",
```
Comment (nathipg) [inline files/jsondata/L10n-SVG.json:96]:
```suggestion
    "pt-BR": "Qualquer número de elementos de animação, elementos descritivos, elementos {{SVGElement(\"mask\")}}, {{SVGElement(\"clipPath\")}}, {{SVGElement(\"marker\")}}, {{SVGElement(\"script\")}}, e {{SVGElement(\"style\")}}, juntamente com zero ou um elemento {{SVGElement(\"meshgradient\")}}.",
```

**Review by nathipg — CHANGES_REQUESTED** [review]
Thank you! I made a suggestion, could you look it, please?

**Comment by nathipg** [comment]
@mdn/yari-content-pt-br OK

**Review by nathipg — APPROVED** [review]

**Review by wbamberg — APPROVED** [review]
👍 looks great, thank you!

**Pull request merged**